sdk/ml/azure-ai-ml/azure/ai/ml/entities/_registry/util.py

```diff
@@ -0,0 +1,16 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+# pylint:disable=protected-access,no-else-return
+
+from azure.ai.ml._restclient.v2022_10_01_preview.models import (
+    ArmResourceId as RestArmResourceId,
+    StorageAccountDetails as RestStorageAccountDetails,
+    UserCreatedStorageAccount as RestUserCreatedStorageAccount)
+
+
+def make_rest_user_storage_from_id(*, user_id: str) -> RestStorageAccountDetails:
```

Comment:
Will this be exposed, if not reccomendation would be to start with "_", to make it private in pythonic way